src/Microsoft.Azure.WebJobs.Host/Scale/TargetScalerDescriptor.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+
+    /// <summary>
+    /// Metadata descriptor for an <see cref="ITargetScaler"/>.
+    /// </summary>
+    public class TargetScalerDescriptor
+    {
+        public TargetScalerDescriptor(string id, string functionId)
+        {
+            Id = id;
+            FunctionId = functionId;
+        }
+
+        /// <summary>
+        /// Gets the unique ID for the monitor.
```

Comment:
You're referencing monitors here which doesn't apply. I don't think we need this ID property at all. Monitors needed a fully formed ID for persistence, which we don't need for target scalers?

Reply:
I am using the "ID" to "merge" scale monitors and target scalers in function host as we register everything in webjobs sdk - if target scaler is defined and TBS is enabled we do not need to use scale monitor.